docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Client Sessions and Robot Networking"
+linkTitle: "Network Disconnection"
+weight: 20
+type: "docs"
+description: "How a Viam robot handles losing its connection to the internet."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+What happens when your robot loses its connection to the internet over a LAN or WAN and can no longer communicate with clients through [the Viam app](https://app.viam.com).
+
+## How does `viam-server` work if your robot loses its network connection?
+
+If the robot loses its connection to the internet, `viam-server` will timeout and end any current client [*sessions*](/program/apis/sessions/) on this robot.
```

Comment:
Try not to start sentences with interrogative words - it somewhat describes the content but it is not an introduction to the topic. How about this:

```suggestion
When a robot loses its connection over LAN or WAN, it can no longer communicate with clients through [the Viam app](https://app.viam.com).
`viam-server` will timeout and end any current client [*sessions*](/program/apis/sessions/) on this robot.

```

Reply:
Added!

Reply:
Made one change-- added `SessionsClient will end` because I believe that API is doing the pruning not `viam-server`, @Otterverse should probably fact-check

Reply:
Can you ask him on that

Reply:
To clarify, sessions are tracked in viam-server itself. When the client connection is interrupted/disconnected, the client is no longer able to to send heartbeats. The viam-server will end the session after a timeout period where no heartbeat is received, it then cancels operations that belong to that session.

Reply:
Ok, thank you so much for the explanation, will change to @npentrel suggestion -- sorry for the confusion